.github/renovate.json

```diff
@@ -11,5 +11,10 @@
     },
     "prCreation": "not-pending",
     "rangeStrategy": "replace",
-    "stabilityDays": 3
+    "stabilityDays": 3,
+    "github-actions": {
+        "fileMatch": [
+            "^ci\\/.*/[^/]+\\.ya?ml$"
```

Comment:
Does this regex also match this file https://github.com/rust-lang/rustup/blob/master/ci/cirrus-templates/freebsd.yaml?

Reply:
yes. Is that a problem?

Reply:
This file is not for GitHub Actions. I don't know whether it makes sense to match it.